pkg/cloud/vsphere/actuators/cluster/actuator.go

```diff
@@ -72,6 +77,28 @@ func (a *Actuator) Reconcile(cluster *clusterv1.Cluster) (opErr error) {
 		return errors.Wrapf(err, "unable to reconcile certs while reconciling cluster %q", ctx)
 	}
 
+	// sync kubeconfig secrets for cluster
+	kubeConfig, err := a.GetKubeConfig(cluster, nil)
+	if err != nil {
+		return errors.Wrapf(err, "unable to get kubeconfig for cluster %q", ctx)
+	}
+
+	secret := &apiv1.Secret{
```

Comment:
Hi @andrewsykim,

I was about to say the same thing regarding `secret` and descriptive, but maybe consider refactoring this entire block into a function `reconcileTargetClusterKubeConfigSecret` or some such thing. That way everything inside is implicitly about the target cluster, and it keeps the outer `Reconcile` function neat and tidy.